en/airframes/airframe_reference.md

```diff
@@ -401,6 +406,25 @@ This page lists all supported airframes and types including
 </tr>
 </tbody></table>
 
+### Quadrotor h
```

Comment:
This is consistent, but should it be capital "H" reflecting the shape?

Reply:
I was going for consistency, but you're right, H makes more sense.